File: komock-core/src/main/kotlin/ua/com/lavi/komock/engine/handler/callback/EmptyCallbackHandlerImpl.java
package ua.com.lavi.komock.engine.handler.callback;

import org.jetbrains.annotations.NotNull;
import ua.com.lavi.komock.engine.model.Request;
import ua.com.lavi.komock.engine.model.Response;

public class EmptyCallbackHandlerImpl implements CallbackHandler {

    @Override
    public void handle(@NotNull Request request, @NotNull Response response) {
        // nothing to do
    }
}

File: komock-core/src/main/kotlin/ua/com/lavi/komock/engine/handler/callback/EmptyCallbackHandlerImpl.kt
package ua.com.lavi.komock.engine.handler.callback

import ua.com.lavi.komock.engine.model.Request
import ua.com.lavi.komock.engine.model.Response

/**
 * Created by Oleksandr Loushkin
 */

class EmptyCallbackHandlerImpl : CallbackHandler {

    override fun handle(request: Request, response: Response) {
        // nothing to do
    }
}
